mock/src/components/Commands.ts

```diff
@@ -0,0 +1,106 @@
+import { REPLFunction } from "./REPLFunction";
+import { Dispatch, SetStateAction, useState } from "react";
+import { mockedDataMap, mockedResultsMap } from "./mockedJson";
+import { HistoryEntry } from "./REPL";
+
+// An export that REPLInput can use to map certain commands to functions
+export const commandMap = new Map<string, REPLFunction>();
```

Comment:
Using this map of strings of the command name to the REPLFunction for the command is a great strategy pattern for making things easy to add!